js/card.js

```diff
@@ -0,0 +1,66 @@
+import { createAds } from './data.js';
+
+const typeKey = {
+  palace: 'Дворец',
+  flat: 'Квартира',
+  house: 'Дом',
+  bungalow: 'Бунгало',
+  hotel: 'Отель'
+};
+
+const mapCanvas = document.querySelector('#map-canvas');
+
+const cardTemplate = document.querySelector('#card')
+  .content
+  .querySelector('.popup');
+
+const similarAds = createAds(1);
+
+const similarAdsFragment = document.createDocumentFragment();
+
+similarAds.forEach(({ author, offer }) => {
+  const cardElement = cardTemplate.cloneNode(true);
+
+  cardElement.querySelector('.popup__avatar').src = author.avatar;
+  cardElement.querySelector('.popup__title').textContent = offer.title;
+  cardElement.querySelector('.popup__text--address').textContent = offer.address;
+  cardElement.querySelector('.popup__text--sum').textContent = `${offer.price} `;
+  cardElement.querySelector('.popup__type').textContent = typeKey[offer.type];
+  cardElement.querySelector('.popup__text--capacity').textContent = `${offer.rooms} комнаты для ${offer.guests} гостей.`;
+  cardElement.querySelector('.popup__text--time').textContent = `Заезд после ${offer.checkin}, выезд до ${offer.checkout}.`;
+
+  const description = cardElement.querySelector('.popup__description');
+  if (offer.description) {
+    description.textContent = offer.description;
+  }
+  else {
+    description.classList.add('hidden');
+  }
```

Comment:
Форматировать лучше так
```suggestion
  } else {
    description.classList.add('hidden');
  }
```